scripts/in_container/verify_providers.py

```diff
@@ -725,6 +740,11 @@ def run_provider_discovery():
         # But the executors command is not available till airflow-2-7-0
         console.print("[bright_blue]List all executors[/]\n")
         subprocess.run(["airflow", "providers", "executors"], check=True)
+    if packaging.version.parse(airflow.version.version) >= packaging.version.parse("2.7.0.dev0"):
```

Comment:
This will only come in 2.8.0 as we probably won't merge it to 2.7

Reply:
Okay. I'll change the version check.